packages/backend-modules/documents/graphql/schema-types.js

```diff
@@ -137,13 +137,13 @@ type Meta {
   sendAsEmail: Boolean
 }
 
-enum DocumentType {
+enum DocumenSchematType {
```

Comment:
```suggestion
enum DocumentSchemaType {
```
Typo :) (bei den anderen auch)